### PR TITLE
Extend JsonEditor with additional controls and forms and add workflows in particular

### PR DESCRIPTION
**Pull request by fiskus**
* Added React context provider where we can set elements for using inside JSONEditor Column
* Added WorkflowsToolbar in particular, that can create additional fields for new workflow using form
* Added components/Form, that can be used for elements for react-final-form. We already use more or less similar but not identical fields in Admin forms and PackageCreate forms, it's easy to unify them

Screenshot from 2022-10-26 11-56-49
Screenshot from 2022-10-26 11-56-59
Screenshot from 2022-10-26 11-57-29

- [x] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: catalog/app/components/FileEditor/QuiltConfigEditor/WorkflowsToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+      setOpen(false)
+    },
+    [onChange],
+  )
+
+  return (
+    <>
+      <Button onClick={() => setOpen(true)} />
+      <Popup
+        bucket={bucket || 'fiskus-sandbox-dev'}
```
Comment (nl0) [inline catalog/app/components/FileEditor/QuiltConfigEditor/WorkflowsToolbar.tsx:246]: are you hardcoding `fiskus-sandbox-dev` bucket intentionally?

File: catalog/app/components/Form/Field.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+})
+
+interface FieldProps {
+  error?: string
+  helperText?: React.ReactNode
+  input: RF.FieldInputProps<string>
+  validating?: boolean
+}
+
+export default function Field({
```
Comment (nl0) [inline catalog/app/components/Form/Field.tsx:19]: "Field" seems too generic imo, especially considering that in fact it's a TextField under the hood

**Review by nl0 — APPROVED** [review]

File: catalog/app/components/FileEditor/QuiltConfigEditor/WorkflowsToolbar.tsx (hunk 1 above)
Comment (fiskus) [reply catalog/app/components/FileEditor/QuiltConfigEditor/WorkflowsToolbar.tsx]: Unintentional, while I was debugging in `/storybook` where is no `bucket`

File: catalog/app/components/Form/Field.tsx (hunk 2 above)
Comment (fiskus) [reply catalog/app/components/Form/Field.tsx]: Renamed to `TextField`

**Review by nl0 — APPROVED** [review]

**Pull request merged**